draft-ietf-quic-tls.md

```diff
@@ -1538,9 +1538,9 @@ discarded.
 
 ## Limits on Key Usage
```

Comment:
Like the tweak.  What do you think about the heading?

Reply:
I originally left it as "Limits on Key Usage" for consistency with RFC8446, though I guess that's not the best rationale. I'm fine with the suggestion.